lib/Client.js

```diff
@@ -682,7 +683,7 @@ class Client extends EventEmitter {
     executeSlackWebhook(webhookID, token, options) {
         const wait = !!options.wait;
         options.wait = undefined;
-        return this.requestHandler.request("POST", (token ? Endpoints.WEBHOOK_TOKEN_SLACK(webhookID, token) : Endpoints.WEBHOOK_SLACK(webhookID)) + (wait ? "?wait=true" : ""), !token, options);
+        return this.requestHandler.request("POST", Endpoints.WEBHOOK_TOKEN_SLACK(webhookID, token) + (wait ? "?wait=true" : ""), false, options);
```

Comment:
Does this one not support auth?

Reply:
To trigger this endpoint, you need to send slack-formatted data so I assumed auth was just not really necessary. We can probably add the option for auth anyway but I didn't think it was of any use